genome_spot/_version.py

```diff
@@ -1 +1 @@
-__version__ = "0.0.0"
+__version__ = "1.0.0"
```

Comment:
Since this is a bug fix, the patch version should be updated => 0.0.1.

1.0.0 would imply a breaking change.

Reply:
I've been informed it should have been 1.0.0 from the start.